tutorials/proto_rename/renaming-proto.md

```diff
@@ -0,0 +1,86 @@
+# Renaming proto fields
+
+Changing a field name will not affect protobuf encoding or compatibility between applications that use proto definitions which differ only by field names.
+The binary protobuf encoding is based on tag numbers, so that is what you need to preserve.
+ 
+## Contents of this tutorial  
+- **person.proto** It's a proto definition. We will rename some fields in next steps and see what will happen.
```

Comment:
Can we instead have a link to open the editor in this file?
You can see the other tutorial for an example.

Reply:
Done.